src/utils.js

```diff
@@ -1,6 +1,9 @@
 const clone = require('safe-clone-deep');
+const pLimit = require('p-limit');
 const configs = require('./configs');
 
+const debugDestinationExistsQueue = pLimit(1);
```

Comment:
What do you think of renaming?

```suggestion
const sequential = pLimit(1);
```

current variable name confused me at first